CONTRIBUTING.md

```diff
@@ -1,178 +1,127 @@
 
-```md
-- [ ] CHANGELOG.md updated
-- [ ] Rebased/mergable
-- [ ] Tests pass
-- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
+```bash
+gvm install go1.8.1
+gvm use go1.8.1 --default
 ```
 
-Signing the CLA
----------------
+#### Version Control Systems
 
-If you are going to be contributing back to InfluxDB please take a
-second to sign our CLA, which can be found
-[on our website](https://influxdata.com/community/cla/).
+Go has the ability to fetch remote packages via revision control systems using the `go get` tool.  Before using `go get`,be sure to install the following revision control software packages on your system:
 
-Installing Go
--------------
-InfluxDB requires Go 1.8.1
+* [Install Git](http://git-scm.com/book/en/Getting-Started-Installing-Git)
+* [Install Mercurial](http://mercurial.selenic.com/wiki/Download)
 
-At InfluxDB we find gvm, a Go version manager, useful for installing Go. For instructions
-on how to install it see [the gvm page on github](https://github.com/moovweb/gvm).
+On Mac or Linux these packages can be installed using package managers such as [brew](http://linuxbrew.sh/), [apt](https://wiki.debian.org/Apt), [pacman](https://wiki.archlinux.org/index.php/Pacman) etc,
 
-After installing gvm you can install and set the default go version by
-running the following:
+#### GNU Make
 
-    gvm install go1.8.1
-    gvm use go1.8.1 --default
+[GNU Make](https://www.gnu.org/software/make) is used for automating the installation of tooling, building packages and running tests, it's commonly used in conjunction with Golang projects  for these tasks.
 
-Installing GDM
--------------
-InfluxDB uses [gdm](https://github.com/sparrc/gdm) to manage dependencies.  Install it by running the following:
+Nearly all (semi-)compliant POSIX operating systems (Mac, Linux) ship with `make` installed out of the box.
 
-    go get github.com/sparrc/gdm
+#### Getting the source
 
-Revision Control Systems
--------------
-Go has the ability to import remote packages via revision control systems with the `go get` command.  To ensure that you can retrieve any remote package, be sure to install the following rcs software to your system.
-Currently the project only depends on `git` and `mercurial`.
-
-* [Install Git](http://git-scm.com/book/en/Getting-Started-Installing-Git)
-* [Install Mercurial](http://mercurial.selenic.com/wiki/Download)
-
-Getting the source
-------
-Setup the project structure and fetch the repo like so:
+The best way to get the source is using `go get`, see the [go get docs](https://golang.org/cmd/go/) to learn more about this process:
 
 ```bash
-    mkdir $HOME/gocodez
-    export GOPATH=$HOME/gocodez
-    go get github.com/influxdata/influxdb
+go get github.com/influxdata/influxdb
 ```
 
-You can add the line `export GOPATH=$HOME/gocodez` to your bash/zsh file to be set for every shell instead of having to manually run it everytime.
+#### Working with a Fork
```

Comment:
I don't think this is the best way contributors can work with a fork of the project. For one thing it can get tricky or annoying with imports, if their editors/IDEs try to use their fork's location in the import paths. A better way to work with forks in Go is provided by [Francesc Campoy](http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html). We should probably suggest people do that.